### PR TITLE
Feat/official journal db

### PR DESCRIPTION
**Pull request by Tryggvig**
First version of sequelize db with service

### CODE REVIEW

File: departments.sql
Comment (jonbjarnio) [inline departments.sql:1]: Er þetta á réttum stað?

File: package.json [hunk 1]
```diff
@@ -107,8 +111,9 @@
     "libphonenumber-js": "1.7.53",
     "license-checker": "^25.0.1",
     "lodash": "4.17.21",
+    "mssql": "^10.0.2",
     "nest-winston": "^1.9.4",
-    "next": "13.3.0",
+    "next": "14.0.4",
```
Comment (jonbjarnio) [inline package.json:116]: Uppfærsla á next?

**Review by jonbjarnio — CHANGES_REQUESTED** [review]

File: apps/official-journal-api-export-e2e/project.json
[the remark this answers is not in the document]
Comment (Tryggvig) [reply apps/official-journal-api-export-e2e/project.json:1]: jebb

File: libs/shared/modules/src/journal/journal.service.ts
[the remark this answers is not in the document]
Comment (Tryggvig) [reply libs/shared/modules/src/journal/journal.service.ts:1]: NEVAH

**Review by jonbjarnio — APPROVED** [review]
LGTM 🤝

**Pull request merged**